File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.java
import java.util.*;

class Solution {
    public String[] solution(String myStr) {
       
        myStr = myStr.replace("a", " ").replace("b", " ").replace("c", " ");
        String[] str = myStr.split(" ");
        
        ArrayList<String> list = new ArrayList<>();
        
        for (int i=0; i<str.length; i++) {
            if (!str[i].equals("")) {
                list.add(str[i]);
            }
        }
        
        if (list.size() == 0) {
            list.add("EMPTY");
        }
        
        String[] answer = new String[list.size()];
        int idx=0;
        for (int i=0; i<list.size(); i++) {
            answer[idx++] = list.get(i);
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181862. 세 개의 구분자/세 개의 구분자.py
def solution(myStr):
    answer = []
    
    for i in myStr:
        if i in "abc":
            myStr = myStr.replace(i, ' ')
    
    answer = myStr.split()
    
    if len(answer) > 0:
        answer = myStr.split()
    else:
        answer = ["EMPTY"]
        
    return answer